src/App.vue

```diff
@@ -1,13 +1,17 @@
 <template>
   <HeaderComp />
+  <CarouselComp />
 </template>
 
 <script>
-import HeaderComp from "@components/Header.vue";
+import HeaderComp from "@/components/Header.vue";
```

Comment:
just call it header and carousel

Reply:
can't call because its show error that vue not accept single word component name